docs/en/01_configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuration
+summary: Configuration options, including enabling extraction for DataObjects, managing cached content length, swapping cache backends, and configuring PDF text extraction
+icon: file-text
```

Comment:
That icon doesn't exist in font awesome 5... did you mean `file-alt`?
https://fontawesome.com/v5/search?q=file-text&o=r

Reply:
Changed to 'cog'